cuav/lib/cuav_mosaic.py

```diff
@@ -679,6 +679,9 @@ def mouse_event_view(self, event):
                                              latlon[0], latlon[1])
             dist = "dist %.1f bearing %.1f alt=%.1f shape=%s" % (distance, bearing, image.pos.altitude, image.shape)
         print("-> %s %s %s" % (latlon, image.filename, dist))
+        if hasattr(cuav_util, 'gps_ddtodms'):
+            latlon_dms = cuav_util.gps_ddtodms(latlon)
+            print("-> (%s, %s) %s %s" % (latlon_dms[0], latlon_dms[1], image.filename, dist))
```

Comment:
This would seem to duplicate most of the normal output.  Can't have that with the default output....  perhaps the mosaic needs a state variable to indicate which output to spit out?

Reply:
I think that state is necessary. And I think that it is better for the user to format the output format. At least I want to specify the GPS format.